runtimes/alibi-explain/mlserver_alibi_explain/runtime.py

```diff
@@ -1,7 +1,11 @@
+import json
 from typing import Any, Optional, List, Dict
 
+import orjson
```

Comment:
Now that `orjson` is an optional dependency on MLServer, would it make sense to add it as a required dep of the `mlserver-alibi-explain` package to ensure it's always there? We did a similar thing on `mlserver-alibi-detect` BTW.

Reply:
done

Reply:
done